src/components/PropertiesNode/index.js

```diff
@@ -29,6 +32,23 @@ class PropertiesNode extends Component {
     });
   }
 
+  componentWillUnmount() {
+    this.applyInputChanges();
+  }
+
+  applyInputChanges = () => {
+    const { nodeDescription, inputText } = this.state;
+    const { dispatch, node: { id, description } } = this.props;
```

Comment:
Do not use dispatch directly as a component prop. I hard recommend to use the `mapDispatchToProps` react-redux feature instead. This kind of approach will make your component dependent to a `dispatch` prop when actually what you need is the function/callback which will call the needed action. Your component doesn't need a dispatch function it needs an action callback.